tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -339,6 +339,24 @@ dhcp_relay/test_dhcp_relay.py::test_dhcp_relay_unicast_mac:
     conditions:
       - "'dualtor' in topo_name and release in ['201811', '201911']"
 
+dhcp_relay/test_dhcp_relay_stress.py::test_dhcp_relay_stress:
+  skip:
+    reason: "Need to skip for platform x86_64-8111_32eh_o-r0"
```

Comment:
Could you please remove this Cisco part? It should be skipped by https://github.com/sonic-net/sonic-mgmt/pull/14959/files